packages/eslint-config-ascribe/rules/custom.js

```diff
@@ -128,6 +128,10 @@ module.exports = {
             'exceptions': ['*']
         }
     }],
+    
+    // We don't like semicolons so kill them
+    // http://eslint.org/docs/rules/semi
+    'semi': ['never'],
```

Comment:
yes, we finally reached peak js hipsterism. Likewise, drinking an avolatte while coding js is now mandatory.

![avolatte_collage_0](https://user-images.githubusercontent.com/90316/26877491-6919c8dc-4b8a-11e7-8b07-c497fac0a497.jpg)